### ISSUE TITLE
Support unicode

### ISSUE THREAD

**Issue by Drugoy** [autokey/autokey#283]
Currently both trigger and replacement parts of hotstrings/abbreviations seem to ignore non-ASCII chars.
#114 covers the trigger part, but the same is absolutely valid for replacement part.

Try to make a hotstring/abbreviation that would send `θ` or any other non-ASCII char - those chars will simply get filtered out.

**Comment by luziferius** [comment]
Yeah, the current X11 interface used (XRecord) can mostly only do, what you can do using the currently configured keyboard layout.
Pasting using the clipboard should work to output non-ASCII characters.

There’s currently a promising possible solution for the output side here #255:
If `xdo` proves stable and fast enough (https://github.com/autokey/autokey/issues/255#issuecomment-479770786), it can be used to implement the output side: https://github.com/autokey/autokey/issues/255#issuecomment-493215258

**Comment by EcherArt** [comment]
Pasting using the clipboard does not work with abbreviations containing non-ASCII characters. Try to type in smth using a СУД abbreviation. Please, fix it.

**Comment by josephj11** [comment]
@EcherArt I'm not an AutoKey developer, but I believe that fixing abbreviations (and replacing XRecord) would require substantial modifications to AutoKey. We have a a few great newer developers, but no one who knows our code that well yet since our prior lead developer (luziferius) left.

This is something that should definitely be done, but don't hold your breath. There is currently no lead developer and we have a growing stack of great enhancements/fixes with no one to integrate them into a new release. Small changes are being added to our development branch, but that is alpha code.

If you know any competent Python developers who want to run a project, send them our way!

**Comment by BlueDrink9** [comment]
The problem is with how we interact with X. Relevant code seems to be in `lib/autokey/interface.py`

**Comment by K0stov** [comment]
Was experimenting with abbreviations today for Serbian Cyrillic in order to make a more ergonomic keyboard layout. Hope this gets resolved in the future and that the developers have gained experience—it would make some peoples’ workflow far easier.

**Comment by josephj11** [comment]
This will probably have to be investigated as part of the Wayland upgrade if we get some developers to work on that.
